bundles/config/org.eclipse.smarthome.config.dispatch/src/main/java/org/eclipse/smarthome/config/dispatch/internal/ConfigDispatcher.java

```diff
@@ -65,22 +74,14 @@
  *
  * @author Kai Kreuzer - Initial contribution and API
  * @author Petar Valchev - Added sort by modification time, when configuration files are read
- * @author Ana Dimova - reduce to a single watch thread for all class instances
+ * @author Ana Dimova - Reduce to a single watch thread for all class instances
+ * @author Henning Treu - Delete orphan exclusive configuration from configAdmin
  */
 public class ConfigDispatcher extends AbstractWatchService {
 
-    private static String getPathToWatch() {
-        String progArg = System.getProperty(SERVICEDIR_PROG_ARGUMENT);
-        if (progArg != null) {
-            return ConfigConstants.getConfigFolder() + File.separator + progArg;
-        } else {
-            return ConfigConstants.getConfigFolder() + File.separator + SERVICES_FOLDER;
-        }
-    }
+    private final Logger logger = LoggerFactory.getLogger(ConfigDispatcher.class);
 
-    public ConfigDispatcher() {
-        super(getPathToWatch());
-    }
+    private final Gson GSON = new Gson();

```

Comment:
Looks like a static field, if the name is in capitals...

Reply:
My impression was (just as for the logger) that the instance(s) from ConfigDispatcher are controlled by the OSGi framework and we dont give it static references to help unload/load the bundle during runtime. Is this not the case here?

Reply:
In case I´m right I could change the name to gson just as for the logger ;-)

Reply:
It is. But then you should comply to the Java naming conventions and not use capitals for that field.